src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -27,7 +27,7 @@ public class Hardware {
 	public static final int INDEX_MOTOR_LOWER = 44;
 	public static final int INDEX_MOTOR_UPPER = 45;
 
-	public static final int FEEDER_MOTOR = 46;
+	// public static final int FEEDER_MOTOR = 46;
```

Comment:
It would be good to add a comment to know this change needs to be reverted before merge. 
Ditto for all comments.